src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -174,9 +212,16 @@ export class WorkspaceSavedObjectsClientWrapper {
       attributes: Partial<T>,
       options: SavedObjectsUpdateOptions = {}
     ): Promise<SavedObjectsUpdateResponse<T>> => {
-      if (this.isRelatedToWorkspace(type)) {
-        await this.validateSingleWorkspacePermissions(id, wrapperOptions.request, [
+      const workspacePermitted = await this.validateWorkspaceAttributesPermitted(
+        attributes,
```

Comment:
The attributes is a partial of saved object which may not contain the workspaces attributes. Do we need to get the updated object by id and type first, and than check intersection with attributes.workspaces?

Reply:
I think there is no need to do this. Because we will check whether the workspace attribute exists and whether the workspace attribute is empty, and will not send extra requests. What's your opinion?

Reply:
My mean is we modify a saved object workspace here, but we don't pass the workspace attributes. Would it be a bug if user have workspace permission here, but doesn't pass the workspaces attributes?